examples/aggregator/scripts/docker_push

```diff
@@ -9,8 +9,8 @@ readonly AGGREGATOR_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-aggregator'
 readonly BACKEND_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-aggregator-backend'
 
 declare -a image backend_image
-image="$(find ./bazel-cache -name aggregator_image.tar)"
-backend_image="$(find ./bazel-cache -name aggregator_backend_image.tar)"
+image="$(find -L ./bazel-bin -name aggregator_image.tar)"
```

Comment:
This search doesn't work outside the Docker container.
And it's currently not possible to push Docker images from within the container (you need to set up GCP).

Reply:
I will revert it. The original search did not work on my machine for some reason.

Reply:
It will only work if you run this search after you build an example.
Because consecutive builds invalidate build targets in the cache IIUC

Reply:
It did not work on my machine even after running `build_example` with `-i`. Perhaps I am running a different version of Bazel or my Bazel is configured differently.